### ISSUE TITLE
Playback time issue

### ISSUE THREAD

**Issue by Mteles96** [AlexInCube/AlCoTest#52]
### Who are you? User (who just use bot from discord)/developer/admin (who download all bot stuff to host)?

User

### Description

The music stops on its own, plays for a while and then cuts off. tpi it plays for about 3 or 4 minutes and then suddenly it stops, all the big songs ended without ending.

### Steps To Reproduce

1. Use command
2. /start [link playlist spotify/youtube]
3. Playback starts and the problem starts, in the first songs this happens for 3 to 5 minutes of music.
4. skip to the next one without finishing the song.

### How reliably can you recreate this issue using your steps above?

Always

**Comment by AlexInCube** [comment]
this happens only with youtube? or with spotify too?

**Comment by Mteles96** [comment]
At the moment I was only able to identify this bug through Spotify

**Comment by AlexInCube** [comment]
Where you host your instance of bot? I mean country

**Comment by Mteles96** [comment]
I'm in Brazil, for now I'm hosting on a host here, I confirmed that this bug also happens on YouTube.

**Comment by AlexInCube** [comment]
For Soundcloud or in other services this bug is happens?

**Comment by AlexInCube** [comment]
Also, what are the specs of your VPS?

**Comment by Mteles96** [comment]
including translating all the text from .locale to pt-BR, if you want to add it, I can share it later.

I'll test whether this occurs on other services, but at the moment I've only added the Youtube and Spotify services

**Comment by Mteles96** [comment]
I'm hosting locally via VScode for now, but I'm going to do it on one called Discloud.

OS: win32
CPU: AMD Ryzen 9 5900X 12-Core Processor            
CPU Usage: 5.44%
RAM Usage: 13878.62 MB / 32676.5 MB

**Comment by AlexInCube** [comment]
There are many reasons why song playback may be interrupted. Starting from weak server characteristics, weak internet or some sanctions from the service.

**Comment by AlexInCube** [comment]
Can you host from Docker Desktop?

**Comment by Mteles96** [comment]
> Você pode hospedar a partir do Docker Desktop?

Yes, I can, but I'm still facing some translation problems. I believe it's a local bug, it always comes back to En..

**Comment by Mteles96** [comment]
But I'm going to do it anyway to get a better result.

**Comment by Mteles96** [comment]
Vou realizar os testes nos outros serviços, como o SoundCloud, para verificar se o problema persiste. Assim que tiver os resultados, volto a te informar com os detalhes.

**Comment by Mteles96** [comment]
> > I believe it's a local bug, it always comes back to En..
> 
> Officially, I only support Russian and English.

No problem, only the texts were changed, the rest is the same.

**Comment by AlexInCube** [comment]
If you want add custom locale, you need create new folder in https://github.com/AlexInCube/AlCoTest/tree/master/src/locales, and set BOT_LANGUAGE env variable to the name of the folder.

**Comment by Mteles96** [comment]
> Se você quiser adicionar localidade personalizada, você precisa criar uma nova pasta no https://github.com/AlexInCube/AlCoTest/tree/master/src/locales e definir BOT_LANGUAGE variável env para o nome da pasta.

I did it, exactly this way

**Comment by AlexInCube** [comment]
> > Can you host from Docker Desktop?
> 
> Yes, I can, but I'm still facing some translation problems. I believe it's a local bug, it always comes back to En..

So i don`t know, how it can comeback to En

**Comment by Mteles96** [comment]
> > > Can you host from Docker Desktop?
> > 
> > 
> > Yes, I can, but I'm still facing some translation problems. I believe it's a local bug, it always comes back to En..
> 
> So i don`t know, how it can comeback to En

I believe it's a Docker bug, that's okay

**Comment by AlexInCube** [comment]
> > > > Can you host from Docker Desktop?
> > > 
> > > 
> > > Yes, I can, but I'm still facing some translation problems. I believe it's a local bug, it always comes back to En..
> > 
> > 
> > So i don`t know, how it can comeback to En
> 
> I believe it's a Docker bug, that's okay

When you add localization to Docker, are rebuild the image for your purpose? Because my .sh scripts are downloading image from Docker Hub.

**Comment by Mteles96** [comment]
> > > > > Can you host from Docker Desktop?
> > > > 
> > > > 
> > > > Yes, I can, but I'm still facing some translation problems. I believe it's a local bug, it always comes back to En..
> > > 
> > > 
> > > So i don`t know, how it can comeback to En
> > 
> > 
> > I believe it's a Docker bug, that's okay
> 
> When you add localization to Docker, are rebuild the image for your purpose? Because my .sh scripts are downloading image from Docker Hub.

Yes, when I add the location I rebuild the image to ensure the changes are applied correctly. However, I noticed that your .sh scripts are downloading the image directly from Docker Hub, so it's possible that this image is overwriting my localized version. I will review the process to ensure the right image is used.

**Comment by Wsympson** [comment]
I'm getting the same issue with YouTube, Spotify, and SoundCloud. US based and running on a dedicated Ubuntu server with 1gb up and down Internet. 
![thorium_DHcRFh2qsd](https://github.com/user-attachments/assets/c8fb81a7-bde1-4e46-a681-43c903d27a23)
![thorium_TxsUQtlIvr](https://github.com/user-attachments/assets/1243a529-a774-489e-ad9a-4bdb8c3eaf57)

**Comment by AlexInCube** [comment]
> I'm getting the same issue with YouTube, Spotify, and SoundCloud. US based and running on a dedicated Ubuntu server with 1gb up and down Internet. ![thorium_DHcRFh2qsd](https://private-user-images.githubusercontent.com/32970296/375917975-c8fb81a7-bde1-4e46-a681-43c903d27a23.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg3MzYwNDgsIm5iZiI6MTcyODczNTc0OCwicGF0aCI6Ii8zMjk3MDI5Ni8zNzU5MTc5NzUtYzhmYjgxYTctYmRlMS00ZTQ2LWE2ODEtNDNjOTAzZDI3YTIzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDEyVDEyMjIyOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWNmZDM4MzkyMWRkZDMxNjA1YjIyZjBlYWYyNDg2YWMzNjVmNWQ5MGI3YjFlYTkyN2I1NmJhZDJmZDU3MTRkZGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.6Zcf2rbVVbObwvSZUYrV3P28gYOFhLHUyUu-RLOrRSw) ![thorium_TxsUQtlIvr](https://private-user-images.githubusercontent.com/32970296/375918034-1243a529-a774-489e-ad9a-4bdb8c3eaf57.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg3MzYwNDgsIm5iZiI6MTcyODczNTc0OCwicGF0aCI6Ii8zMjk3MDI5Ni8zNzU5MTgwMzQtMTI0M2E1MjktYTc3NC00ODllLWFkOWEtNGJkYjhjM2VhZjU3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDEyVDEyMjIyOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFjNzg1ZjgxODMzZWY2N2NlOGJmMzFhMjg1Y2Y5NjQzYTY4YzJjMGNlZTc1ZDc0M2VlNmZkYzI1ODc0YjQ0YzQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.LcWjVeZFkQUxv28ocMX0QP_F1KamcltD65qZmxFLWFc)

I don't know why, but at US people are having trouble using the bot. Personally, I use my bot in Ukraine and all services work for me. I don't know how to fix error 403
https://github.com/AlexInCube/AlCoTest/issues/51

**Comment by AlexInCube** [comment]
I read the “help” section in the Distube Discord community, there are a lot of people complaining about the 403 error. And not for the first time, I think I need to rewrite the bot to Lavalink or some other library.

**Comment by Mteles96** [comment]
I think Lavalink would be good.

> I read the “help” section in the Distube Discord community, there are a lot of people complaining about the 403 error. And not for the first time, I think I need to rewrite the bot to Lavalink or some other library.

**Comment by Mteles96** [comment]
> I read the “help” section in the Distube Discord community, there are a lot of people complaining about the 403 error. And not for the first time, I think I need to rewrite the bot to Lavalink or some other library.

I think Lavalink would be good...
